Makefile

```diff
@@ -370,7 +401,22 @@ get-deps-init:
 
 generic-rpm: .generic-rpm-done
 
-dockerfree-all: dockerfree-agent-image generic-rpm
+rpm-done:
+	./scripts/update-version.sh
+	cp packaging/generic-rpm/amazon-ecs-init.spec amazon-ecs-init.spec
+	cp packaging/generic-rpm/ecs.service ecs.service
+	cp packaging/generic-rpm/amazon-ecs-volume-plugin.service amazon-ecs-volume-plugin.service
+	cp packaging/generic-rpm/amazon-ecs-volume-plugin.socket amazon-ecs-volume-plugin.socket
+	tar -czf ./sources.tgz ecs-init scripts
+	test -e SOURCES || ln -s . SOURCES
+	rpmbuild --define "%_topdir $(PWD)" -bb amazon-ecs-init.spec
+	find RPMS/ -type f -exec cp {} . \;
+	touch .rpm-done
```

Comment:
this touch doesn't really do anything anymore unless the make target is split into `.rpm-done` and `rpm-done: .rpm-done` like the integrated ones are